kv/main.go

```diff
@@ -129,7 +131,7 @@ type generator struct {
 func newGenerator(seq *sequence) *generator {
 	return &generator{
 		seq:    seq,
-		rand:   rand.New(rand.NewSource(int64(time.Now().UnixNano()))),
+		rand:   rand.New(rand.NewSource(rand.Int63())),
```

Comment:
if `time.Now().UnixNano()` returns the same timestamp (surely shouldn't on reasonable systems, but Never Trust Clocks™), you end up with multiple `blockers` overlapping, which is unintentional. Seems more legit this way.